ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -1677,6 +1675,16 @@ private enum UnityLayers {
     }
   }
 
+  private void RemoveStockTrajectoriesIfNeeded(CelestialBody celestial) {
+    if (celestial.orbitDriver == null) {
+      return;
+    }
+    celestial.orbitDriver.Renderer.drawMode =
+        main_window_.display_patched_conics
```

Comment:
Do we know why we still have that option?

Reply:
Mostly comparing with stock, for our entertainment.